src/server/coprocessor/endpoint.rs

```diff
@@ -215,20 +215,23 @@ impl BatchRunnable<Task> for Host {
                             continue;
                         }
                     };
-                    let end_point = TiDbEndPoint::new(snap);
+                    let len = reqs.len() as f64;
```

Comment:
seem that we can add two metrics here

1. How many requests in one batch? If most of the time, it is only 1, the change seems un-necessary. 
2. How many ranges one request scans? If the request needs to scan many ranges, do we need to scan concurrently?